stable/graylog/templates/files-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.graylog.serverFiles -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "graylog.fullname" . }}-files
+  labels:
+    app: {{ template "graylog.fullname" . }}
```

Comment:
Do we want to use new labeling ?

Reply:
@maver1ck what do you mean new labels?

Reply:
@KongZ this
https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels

Reply:
@maver1ck I updated with new standard labels. Thanks :)